src/app/shared/utils/consts.ts

```diff
@@ -6,6 +6,10 @@ export let DATASET_TYPE_MAP: any = {
 
 export let DEFAULT_GROWL_LIFE: number = 7000;
 
+export let DEFAULT_ANGULAR_DATE_FORMAT: string = 'dd/MM/yyyy';
+
 export let DEFAULT_PRIME_DATE_FORMAT: string = 'dd/mm/yy';
 
 export let DEFAULT_MOMENT_DATE_FORMAT: string = 'DD/MM/YYYY';
+
+export let AMBIGOUS_DATE_PATTERN: RegExp = /^(\d{1,2}).(\d{1,2}).(\d{4})$/;
```

Comment:
Don't you want to specify the separation character (`/` or `-`). 
I don't think you want something like `2530192018` to match.
Maybe:
```
/^(\d{1,2})[-,\/](\d{1,2})[-,\/](\d{4})$/
````